crates/common/src/lib.rs

```diff
@@ -298,12 +298,18 @@ impl TransactionVersion {
     pub const ONE: Self = Self(H256(hex_literal::hex!(
         "0000000000000000000000000000000000000000000000000000000000000001"
     )));
+    pub const TWO: Self = Self(H256(hex_literal::hex!(
+        "0000000000000000000000000000000000000000000000000000000000000002"
+    )));
```

Comment:
This is pretty much the only usage of `hex_literal` in pathfinder, so I think we can remove it quite easily:
```suggestion
    pub const TWO: Self = Self(H256::from_low_u64_be(2));
```

Reply:
Unfortunately `from_low_u64_be` is non const.

Reply:
Hmm. Really annoying that.